### ISSUE TITLE
request - show error message on input

### ISSUE THREAD

**Issue by donnygunawan93** [quasarframework/quasar#279]
```
<div class="item-content">
    <div class="floating-label">
        <input v-model="login"  required class="full-width" :class="errors.first('login') ? 'has-error' : ''">
        <label>Username or Email</label>
    </div>
    <small><sup class="text-red">{{ errors.first('login') }}</sup></small>
</div>
```
I need to check if "has-error", then print the error below.
Would be great if we can do in a single move like this https://museui.github.io/#/textField
`<input v-model="login"  class="full-width" msgError="errors.first('login')">`

**Comment by rstoenescu** [comment]
Yes, good idea. Will work on this. Thanks!

**Comment by rstoenescu** [comment]
Will be available in v0.14
